data/boarding missions.txt

```diff
@@ -23,6 +23,118 @@ mission "Assisting Merchant"
 	# A mission with no destination will not be offered, so give it one.
 	# What the destination is doesn't matter, because you always 'decline' this.
 	destination Earth
+	
+mission "Assisting Pirates (Thief)"
+	assisting
+	repeat
+	to offer
+		random < 12
+	source
+		government Pirate
+	on offer
+		payment -100000
+		conversation
+			`After assisting the <origin> you find that 100000 credits have been removed from your account.`
+				decline
+	destination Greenrock
+
+mission "Assisting Pirates (Setback)"
+	assisting
+	repeat
+	to offer
+		random < 8
+	source
+		government Pirate
+	on offer
+		payment 100
+		conversation
+			`The captain of the <origin> gives you <payment> for "helping him out of a temporary setback."`
+				decline
+	destination Greenrock
+	
+mission "Assisting Syndicate"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Syndicate
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Syndicate cause, the captain gives you <payment>.`
+				decline
+	destination Foundry
+	
+mission "Assisting Oathkeepers"
+	assisting
+	repeat
+	to offer
+		random < 25
+	source
+		government "Navy (Oathkeeper)"
+	on offer
+		payment 7500
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Oathkeepers' cause, the captain gives you <payment>.`
+				decline
+	destination Farpoint
+
+mission "Assisting Navy"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Republic
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Navy's cause, the captain gives you <payment>.`
+				decline
+	
+mission "Assisting Militia"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government Militia
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Militia's cause, the captain gives you <payment>.`
+				decline
+	destination Girtab
+
+mission "Assisting Free Worlds"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government "Free Worlds"
+	on offer
+		payment 5000
+		conversation
+			`Knowing you risked your life to save the <origin> and help the Free Worlds' cause, the captain gives you <payment>.`
+				decline
+	destination Bourne
+
+mission "Assisting Free Worlds (Join)"
```

Comment:
Why keep this one if you've thrown out almost all of the other variants?

Reply:
A hint to the story or more accurately a hint as to what to do.

Reply:
Then you should probably make a branch on the first assisting FW mission, as well as the Navy and Syndicate missions for the sake of forward compatibility.
```
conversation
	branch sides
		has "chosen sides"
	`Knowing you risked your life to save the <origin> and help the Free Worlds' cause, the captain gives you <payment>.`
		decline
	label sides
	`Knowing you risked your life to save the <origin> and help the Free Worlds' cause, the captain gives you <payment>. along with a message saying, "Join the Free Worlds!"`
		decline
```

Reply:
that branch should be `has "FW Commitment: done"` and not `has "chosen sides"`

Reply:
Then if you join the Navy or Syndicate, the assisting Free World ships will constantly have them asking you to join them despite being unable to, and even if you might be their enemy at the time of a cease fire.

Reply:
ah, I read your comment as the captain recognizing the player also helps the Free Worlds outside of just helping him.

For the case of hinging on just can / can't join, then `chosen sides` is indeed sufficient.